Lets Do This/Controllers/LDTRegisterViewController.h

```diff
@@ -12,6 +12,8 @@
 
 @property (nonatomic, strong) IBOutlet UITextField *emailField;
 @property (nonatomic, strong) IBOutlet UITextField *passwordField;
+@property (weak, nonatomic) IBOutlet UITextField *firstNameField;
+@property (weak, nonatomic) IBOutlet UITextField *lastNameField;
```

Comment:
@ryangrimm I declared these here to keep things consistent, but would it be better practice to move all of these outlets  into the `.m` file?